test_1.sh

```diff
@@ -1,12 +1,13 @@
 #!/bin/bash
 
 set -eu
-set -o pipefail
+# set -o pipefail
```

Comment:
do you remember why you needed this one?

Reply:
I think you told me to remove the flag while we were sitting next to each other - I didn't fully understand why so I commented it, hoping I would remember to either figure out or replace. So if you don't know either, let me replace

Reply:
ah, ok... I had to add it whenever I added `| head -n 10` into pipeline after `find`. You don't need that here AFAIK, hence let's minimize the diff

```suggestion
set -o pipefail
```